SpatialGDK/Source/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -139,8 +131,19 @@ void USpatialActorChannel::Close()
 	Super::Close();
 }
 
-TArray<uint16> USpatialActorChannel::SkipOverChangelistArrays(FObjectReplicator& Replicator)
+bool USpatialActorChannel::IsDynamicArrayHandle(UObject* Object, uint16 Handle)
+{
+	check(ObjectHasReplicator(Object));
+	FObjectReplicator& Replicator = FindOrCreateReplicator(Object).Get();
+	TSharedPtr<FRepLayout>& RepLayout = Replicator.RepLayout;
```

Comment:
Why a reference? won't this break the ref counting?

Reply:
No. This will act like an alias. No object is created or destroyed.